spec/controllers/api_controller_spec.rb

```diff
@@ -282,7 +282,7 @@
         expect(team['leader']).to eq false
         expect(team['friendly_id']).to eq 'wrc'
         expect(team['avatar']['thumb']['url']).to be_a String
-        expect(team['id']).to be_a Numeric
+        expect(team['id']).to be_a String
```

Comment:
It _should_ still be numeric, right?
I mean, the primary key ID of a role object is still an integer in the end...

Reply:
Since in this context the user is signing in as a team-member, the team's ID will be the ID of made-up group, which will be of something like `teams_committees_123`. But once the teams are migrated to groups, this test-case will start failing and that time will change it back to Numeric.

Reply:
There is a small correction. team['id'] was team_member ID earlier, and now it's role ID. Since there are no role ID for teams-committees yet, this will be string. But once I raise the PR for roles migration of teams-committees, this will be changed back to Numeric.